test/puppetlabs/puppetdb/admin_test.clj

```diff
@@ -34,77 +50,40 @@
                     (assoc :certname certname
                            :producer_timestamp (time/now)))
         report (-> (:basic example-reports/reports)
-                   (assoc :certname certname))]
+                   (assoc :certname certname)
+                   tur/munge-example-report-for-storage)]
 
     (svc-utils/call-with-single-quiet-pdb-instance
      (fn []
        (let [query-fn (partial query (tk-app/get-service svc-utils/*server* :PuppetDBServer))
-             submit-command-fn (partial submit-command (tk-app/get-service svc-utils/*server* :PuppetDBCommand))]
-         (is (empty? (query-fn :nodes admin/query-api-version nil nil doall)))
+             submit-command-fn (partial submit-command (tk-app/get-service svc-utils/*server* :PuppetDBCommand))
+             base-url (tu/command-base-url svc-utils/*base-url*)]
+         (is (empty? (query-fn :nodes :v4 nil nil doall)))
```

Comment:
similar here -- we actually have query-api-version still defined in export (maybe there's a better place for it) but we've switched to hardcoding :v4